### PR TITLE
feat: Change name of Preferences > Filters > Tabs and move them to Account Preferences(#3536)

### LINKED ISSUE #3536: Change name of Preferences > Filters > Tabs
Open Preferences and there's a section with the heading "Filters", and the single entry "Tabs".

This is confusing, and I've just figured out why.

1. These aren't filters in the normal Mastodon sense of the word. 
2. These aren't really "tab" options, they're timeline options.

Proposed concrete change:

1. Change the section title from "Filters" to "Per-timeline preferences"
2. Since the current "Tabs" entry is only for the Home timeline, change that to "Home timeline".

In the future, if other per-timeline options are needed (for example a "Show unread direct message count") then entries for those timelines can be added as additional sub-preference screens under here.

* * * *
- Tusky Version: 21

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

### PR DESCRIPTION
**Pull request by sanao1006**
# Overview
In the previous code, when you open preferences, there is a section headed "Filters" with a section called "Tabs"

This is confusing.

# Changes
- Change the section title from "Filters" to "Per-timeline preferences."
- Change the current "Tabs" section to "Home timeline" since it is only for home timelines

# Screenshots
account preference screen | detail screen
:--: | :--:
|<image src="https://github.com/tuskyapp/Tusky/assets/62137820/12694f24-b7e3-4ba3-90f5-53740e9c4269" width="250" />|<image src="https://github.com/tuskyapp/Tusky/assets/62137820/796e9ac1-76d6-43ef-a087-a1cd2d899ef8" width="250" />

# Note
- Maybe string resources should have a new property? (for translation)

# Related link
 Fixes #3536

### CODE REVIEW

File: app/src/main/java/com/keylesspalace/tusky/settings/SettingsConstants.kt [hunk 1]
```diff
@@ -45,7 +45,7 @@ enum class AppTheme(val value: String) {
  *
  * - Adding a new preference that does not change the interpretation of an existing preference
  */
-const val SCHEMA_VERSION = 2023082301
+const val SCHEMA_VERSION = 2023112001
```
Comment (sanao1006) [inline app/src/main/java/com/keylesspalace/tusky/settings/SettingsConstants.kt:48]: I don't know if this way of updating SharedPreferences is correct.

So, please point out if I am wrong!

**Comment by sanao1006** [comment]
Are you possibly forgetting about this PR?
I appreciate if you can give me any response!

File: app/src/main/java/com/keylesspalace/tusky/settings/SettingsConstants.kt (hunk 1 above)
Comment (connyduck) [reply app/src/main/java/com/keylesspalace/tusky/settings/SettingsConstants.kt:48]: You did it correctly

**Review by connyduck — COMMENTED** [review]
Very nice, thank you.

"Per-timeline preferences" is now shown twice on the account preferences which is a bit confusing. I think one of these should be renamed. Suggestion: Call the new one at the botton "local filters".

**Comment by sanao1006** [comment]
Sorry, I didn't notice the "Per-timeline preferences" appearing at the top.

I thought the "Per-timeline preferences" at the top might be unnecessary rather than renaming it.

**Comment by sanao1006** [comment]
Sorry, I retract my statement above. 
It is true that it would be better to rename it as you said!

**Comment by sanao1006** [comment]
@connyduck 
How about setting the new one to "Per-timeline preferences" and the old one back to "Filters"?

**Comment by sanao1006** [comment]
Fixed as shown in the image above.

**Review by connyduck — APPROVED** [review]

**Comment by mcclure** [comment]
Can you explain to me the meaning of the second commit, "[update: Remove unnecessary sharedPreferences](https://github.com/tuskyapp/Tusky/pull/4115/commits/6d537dccbdc051a18047b2297b31ca0c0dc08c8e)"? Were these fields already or are they made irrelevant by this patch changing them to per-account values? Thanks.

Also, I have put up a PR #4128 , if that is accepted this PR will need to change its version number to acommodate it.

**Comment by sanao1006** [comment]
> are they made irrelevant by this patch changing them to per-account values?

Yes, exactly.

> Also, I have put up a PR https://github.com/tuskyapp/Tusky/pull/4128 , if that is accepted this PR will need to change its version number to acommodate it.

In other words, this PR needs to update its DB version to 56 instead of 55, is that right?

**Comment by mcclure** [comment]
Thanks for explaining.

> > Also, I have put up a PR #4128 , if that is accepted this PR will need to change its version number to acommodate it.
> 
> In other words, this PR needs to update its DB version to 56 instead of 55, is that right?

If the PR is accepted. (But there were no objections either here or on Matrix over a while so I guess it is getting accepted?)

**Comment by mcclure** [comment]
Updated PR with 55->56 change as discussed. I tested this builds but did not test running it or performing an upgrade.

Sanao, this will probably cause your local copy to crash until you do an uninstall/reinstall cycle.

**Pull request merged**